tests/bdd/flex/bbox.feature

```diff
@@ -71,10 +71,55 @@ Feature: Test get_bbox() function
 
         Then table osm2pgsql_test_highways contains exactly
             | way_id | tags->'highway' | min_x | max_x | min_y | max_y | ST_AsText(geom) |
-            | 20     | primary         | 20.0  | 20.1  | 10.0  | 10.1  | <geometry> |
+            | 20     | primary         | 20.0  | 20.1  | 10.0  | 10.1  | <geometry>      |
 
         Examples:
-            | projection | geometry |
-            | 4326       | 20 10.1,20.1 10.1,20.1 10 |
+            | projection | geometry                                                    |
+            | 4326       | 20 10.1,20.1 10.1,20.1 10                                   |
             | 3857       | 2226389.8 1130195.4,2237521.8 1130195.4,2237521.8 1118890.0 |
 
+    Scenario Outline: for relations
+        Given the 0.1 grid with origin 20.0 10.1
+            | 10 | 11 |
+            |    | 12 |
+        And the OSM data
+            """
+            w20 v1 dV Nn10,n11
+            w21 v1 dV Nn11,n12
+            r30 v1 dV Ttype=route,route=bus Mw20@@
+            r31 v1 dV Ttype=route,route=bus Mw20@,w21@
+            """
+        And the lua style
+            """
+            local rels = osm2pgsql.define_relation_table('osm2pgsql_test_routes', {
+                { column = 'tags',  type = 'hstore' },
```

Comment:
I already noticed when porting to BDD: do we need to keep the tags column? It's not relevant to the tests.

Reply:
I have removed all the "tags" columns from the bbox tags.